Sources/Receipt/ListReceiptTableViewController.swift

```diff
@@ -21,7 +21,7 @@ import UIKit
 /// Lists the user's transaction history
 ///
 /// The user can click a receipt in the list for more information
-public final class ListReceiptViewController: UITableViewController {
+public final class ListReceiptTableViewController: UITableViewController {
```

Comment:
We cannot remove public because it will give this compilation error
```Method cannot be declared public because its result uses an internal type```
for this ```
public func listTransferMethodTableViewController() -> ListTransferMethodTableViewController {
        return ListTransferMethodTableViewController()
    }
```